docs/guides/testing-single-file-components-with-mocha-webpack.md

```diff
@@ -148,13 +148,15 @@ Create a file in `src` named `Counter.vue`:
 And create a test file named `test/Counter.spec.js` with the following code:
 
 ```js
+import Vue from 'vue';
```

Comment:
Thank you for the PR! Prettier is complaining about this semicolon. `yarn run format` will fix it.

Reply:
You can also use `wrapper.vm.$nextTick()`, but this could be a bit more future proof, as nextTick will no longer be avail on the vm in V3...